README.md

```diff
@@ -196,6 +198,43 @@ const view = (state, actions) =>
 
 It is important to understand that the result of the view is a new Virtual DOM. It may seem wasteful to throw away the old Virtual DOM and re-create it entirely on every update — not to mention the fact that at any one time, Hyperapp is keeping two Virtual DOM trees in memory, but as it turns out, browsers can create hundreds of thousands of objects very quickly. On the other hand, modifying the DOM is several orders of magnitude more expensive.
 
+## Mounting
+
+> **Work In Progress**: This section is not perfect and may be incomplete. Please help.
```

Comment:
Here's some alternative wording:

In order to start your app, call

```js
app(state, actions, view, container)
```

where `container` is an element in your DOM. Hyperapp will produce a DOM element from the vnode returned by your `view`, and add it to the children of `container`.

If there are already children in `container`, the app's root-element will be inserted above them, but they will be otherwise left untouched.